src/sentry/tasks/derive_code_mappings.py

```diff
@@ -70,7 +70,7 @@ def derive_code_mappings(
 
     try:
         with lock.acquire():
-            trees = installation.get_trees_for_org(3600 * 3)
```

Comment:
See comment in here:
https://github.com/getsentry/sentry/pull/42082/files#diff-03b76d14306edb8c166bdf3da6838ff578a65e108e60b75a3e8dfe772ba4d73eR78